src/content/en/tools/workbox/v3/modules/workbox-cli.md

```diff
@@ -0,0 +1,161 @@
+project_path: /web/tools/workbox/v3/_project.yaml
+book_path: /web/tools/workbox/v3/_book.yaml
+description: The module guide for workbox-cli.
+
+{# wf_updated_on: 2017-11-27 #}
+{# wf_published_on: 2017-11-27 #}
+
+# Workbox CLI  {: .page-title }
+
+## What's the Workbox CLI?
+
+The Workbox command line interface (contained in the the
+`workbox-cli` package) consists of a Node.js program called workbox that
+can be run from a Windows, MacOS, of UNIX-compatible command line
+environment. Under the hood, workbox-cli wraps the  workbox-build module,
+and provides an easy way of integrating Workbox into a command line build
+process, with flexible configurations.
+
+## Install the CLI
+
+Installing the CLI is simple, just run the following command in your
+terminal.
+
+* {NPM}
+
+    ```
```

Comment:
Check out this [syntax for terminal code](https://developers.google.com/internal/publishing/reference/styles#terminal_code)

Also, should it be indented when you're wrapping in triple back ticks?